ligand/templates/target_details.html

```diff
@@ -434,20 +520,61 @@
           table_affinity.draw();
         } );
 
+        function reset_hidden_target_details_affinity() {
+          var table_affinity = target_details_table_affinity;
+  
+          let affinity_length = table_affinity.columns()[0].length;
+          let affinity_columns = Array.from(new Array(affinity_length - 1), (x, i) => i);
+          table_affinity.columns(affinity_columns).visible(true, false);
+          table_affinity.draw();
+        }
+        function reset_hidden_target_details_potency() {
+          var table_potency = target_details_table_potency;
+
+          let potency_length = table_potency.columns()[0].length;
+          let potency_columns = Array.from(new Array(potency_length - 1), (x, i) => i);
+          table_potency.columns(potency_columns).visible(true, false);
+          table_potency.draw();
+        }
+
+
+        setTimeout(function () {
+          $("a[href='#affinity']").trigger('click');
+          setTimeout(function () {
+            reset_hidden_target_details_affinity();
+            setTimeout(function () {
+              $("a[href='#potency']").trigger('click');
+              reset_hidden_target_details_potency();
+            },0);
+          },0);
+        },0);
+        
+        
+        
+
         $('#csv_btn_potency').click(function () {
+            // TODO: redactor in a single fucntion shared by csv_btn_potency and csv_btn_potency affinity
             var checked_data = table_potency.rows('.alt_selected').data();
             var checked_data = table_potency.rows('.alt_selected').data();
             var csv_data = [];
-            csv_data.push('ChEMBL ID;Receptor;Species;Purchasable;No. records;Assay type;Min.;Average;Max.;Unit;Mol. weight;Rot. Bonds;H don;H acc;LogP;Smiles')
+
+            {% if mode == 'extended' %}
+              csv_data.push('Lig. Name;GPCRdb ID;#Vendors;Ref. Ligand;Fold selectivity (Potency);# Tested GPCRs;Species;p-value (-log);Value type;Activ. relation;Activ. value;Assay type;Assay descript.;Source; Mol. weight;Rot. Bonds;H don;H acc;LogP;Smiles;DOI')
+            {% endif %}
+            {% if mode == 'compact' %}
+              csv_data.push('Lig. Name;GPCRdb ID;Lig. type;Ref. Ligand;Fold selectivity (Potency);# Tested GPCRs;{% if ligand_query %}Receptor (Gtp);Receptor (UniProt);{% endif %}Species;{% if ligand_query %}Class;Family;{% endif %}#Vendors;Assay type;#Records;Min.;Average;Max.;Activ. Value;Source;Mol. weight;Rot. Bonds;H don;H acc;LogP;Smiles')
+            {% endif %}
             for (i = 0; i < checked_data.length; i++) {
                 var csv_row = []
                 for (j = 1; j < checked_data[i].length; j++) {
                     var div = document.createElement("div");
                     div.innerHTML = checked_data[i][j];
                     if (typeof div.innerText !== "undefined") {
-                        csv_row.push(div.innerText.replace(/\s+/g, ''));
+                        //csv_row.push(div.innerText.replace(/\s+/g, ''));
```

Comment:
please remove commented line

Reply:
Solved in [bb9beb8](https://github.com/protwis/protwis/pull/550/commits/bb9beb8c3231460510e199f66a17a1c512c6c8e0).